contents/handbook/company/offsites.md

```diff
@@ -57,6 +58,8 @@ Ideas for the agenda:
 - Hackathon - try to leave 2 days for this, and most importantly avoid sessions interrupting hacking
 - Even some regular work on ongoing challenging projects - this is the best time for exchanging knowledge!
 
+> When organising a hackathon: Participation should be _very_ strongly encouraged but not mandatory if it does not make sense for some attendees. It is more important that people are fully available and interested in particpating. Given the offsite is an opportunity to work together there should be no teams of one. This extends beyond the formation of teams and into the hackathon itself in cases where there is team switching.
```

Comment:
```suggestion
> Don't run a hackathon during an onboarding offsite. Other offsites normally do have a hackathon. Participation should be _very_ strongly encouraged but not mandatory - if not everyone is taking part make sure that working spaces are available to accommodate the different styles of work. It is super important that people taking part are fully available and focussed on participating. Given the offsite is an opportunity to work together there should be no teams of one. This extends beyond the formation of teams and into the hackathon itself in cases where there is team switching.
```

Reply:
Every non-onboarding offsite I've been to has had a hackathon. I think it's top banana